kotlinpoet/src/main/java/com/squareup/kotlinpoet/FileSpec.kt

```diff
@@ -398,6 +413,28 @@ public class FileSpec private constructor(
       memberImports += Import(memberName.canonicalName, `as`)
     }
 
+    /** Adds a default import for the given [packageName]. */
+    public fun addDefaultPackageImport(packageName: String): Builder = apply {
+      defaultImports += packageName
+    }
+
+    /**
+     * Adds Kotlin's standard default package imports as described
+     * [here](https://kotlinlang.org/docs/packages.html#default-imports).
+     */
+    public fun addKotlinDefaultImports(
+      includeJvm: Boolean = true,
```

Comment:
Should we use "java" instead of "jvm" in naming? "jvm" might suggest it refers to "kotlin.jvm" imports.

Reply:
It does, technically. These are the imports listed under the jvm subsection linked above. It includes java.lang and kotlin.jvm